server/main.go

```diff
@@ -6,23 +6,36 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"time"
 )
 
 const (
 	exitFail = 1
 )
 
 func main() {
-	if err := run(os.Args); err != nil {
+	if err := run(); err != nil {
 		fmt.Fprintln(os.Stderr, err)
```

Comment:
we use log.Fatal usually

Reply:
@Ntr0 it's one of those things I picked up along the way and I was never really happy with log.Fatal (imo its not the role of the logging package to do that).
In the client I use slog and there they do not provide Fatal anymore.

Reply:
btw I will switch to slog in the server as well in a future PR